packages/table/UPGRADE.md

```diff
@@ -0,0 +1,80 @@
+# v10 to v11
+
+v11 introduces a large set of new features including virtualized scrolling and sticky header rows. For a functional overview, refer to the [project brief](https://docs.google.com/document/u/1/d/1AaZfYAGi9MCxU-cutWovDwTl_4jViUP34QwMFiWMSxU/edit).
+
+## Overview
```

Comment:
Should we mention how to migrate from v10 to v11? Maybe two code blocks that render the same table in each version?

Reply:
Could be useful, but I think it might be needlessly verbose for an upgrade guide. There are READMEs for both versions and examples of both versions on SB, so I feel like this wouldn't be necessary

Reply:
That's fair, as long as both code samples are easily accessible 👍